packages/teuchos/core/src/Teuchos_TestingHelpers.hpp

```diff
@@ -258,6 +258,28 @@ bool compareFloatingArrays(
   Teuchos::FancyOStream &out
   );
 
+/** \brief Compare if two array objects are the same up to an absolute
+ * difference: elements <tt>a1[i]</tt> and <tt>a2[i]</tt> are considered
+ * the same if <tt>|a1[i]-a2[i]| <= tol</tt>.
+ *
+ * This function works with any two array objects are the same size and have
```

Comment:
```suggestion
 * This function works with any two array objects that are of the same size and have
```

Reply:
Thanks @maartenarnst , I actually took the part about "same size" out completely because it's not really a precondition - you can pass different sized a1,a2 and it will correctly return false in that case.